src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -183,6 +183,12 @@ def _validate_and_store_performance_problem(self):
             "consecutive_count_threshold"
         )
 
+        exceeds_span_duration_threshold = all(
+            get_span_duration(span).total_seconds() * 1000
+            >= self.settings.get("span_duration_threshold")
+            for span in self.consecutive_http_spans
+        )
```

Comment:
I added `test_does_not_detect_consecutive_http_issue_with_low_duration_spans` test for that case @k-fish. So all conditions hold except that the individual span durations are below the threshold.

Reply:
Ah no I mean, we should test a transaction that:
- Setup a set of problematic spans (6x501ms) that meet the criteria for detection
--  Assert gets detected
- Add a span of `499ms` consecutively after the last span
-- Assert still gets detected.

Currently I suspect if you add another span consecutively it will break, even if the pattern exists in the transaction.

Reply:
Okok I'll add a test for this.

Reply:
I see good catch @k-fish. We wanna check in visit_span instead.